virttest/utils_stress.py

```diff
@@ -197,3 +200,29 @@ def host_stress_event(self, event):
             else:
                 raise NotImplementedError
             time.sleep(self.itr_sleep_time)
+
+
+def install_stressapptest(vm):
+    """
+    Install stressapptest cmd
+
+    :param vm: the vm to be installed with stressapptest
+    """
+    session = vm.wait_for_login(timeout=360)
+    name = ["git", "gcc", "gcc-c++", "make"]
+    if not utils_package.package_install(name, session, timeout=300):
```

Comment:
Not sure if we should install packages in avocado-vt. It is not recommended in the tp-libvirt as you can see for example in this PR: https://github.com/autotest/tp-libvirt/pull/3741.